### PR TITLE
italian translation added

### PR DESCRIPTION
**Pull request by baggius**
added missing italian translation

### CODE REVIEW

File: pegasus_it.ts [hunk 1]
```diff
@@ -0,0 +1,548 @@
...
+<!DOCTYPE TS>
+<TS version="2.1" language="it_IT">
+<context>
+    <name>FilePicker</name>
+    <message>
+        <location filename="../src/frontend/menu/settings/gamedireditor/FilePicker.qml" line="81"/>
+        <source>Select a collection file below (metadata.pegasus.txt or metadata.txt).
+The directory of the file will then be added to the list.</source>
+        <translation>Seleziona un file collezione (metadata.pegasus.txt o metadata.txt).
+        La cartella del file sara' aggiunta alla lista.</translation>
```
Comment (mmatyas) [inline pegasus_it.ts]: I think these leading spaces shouldn't be there.

File: pegasus_it.ts [hunk 2]
```diff
@@ -0,0 +1,548 @@
...
+        <translation>Indietro</translation>
+    </message>
+</context>
+<context>
+    <name>GamepadRecorder</name>
+    <message>
+        <location filename="../src/frontend/menu/settings/gamepad/GamepadRecorder.qml" line="107"/>
+        <source>Tilt any axis now to record
+(timeout in %1 seconds)</source>
+        <translation>Sposta qualsiasi delle assi adesso per registrare (timeout in %1 secondi)</translation>
```
Comment (mmatyas) [inline pegasus_it.ts]: Please add a line break before the `(timeout in %1 secondi)`, and in the text below this one too.

File: pegasus_it.ts [hunk 3]
```diff
@@ -0,0 +1,548 @@
...
+    <name>NoGamesError</name>
+    <message>
+        <location filename="../src/frontend/messages/NoGamesError.qml" line="22"/>
+        <source>No games found :(</source>
+        <translation>Nessun gioco trovato :(</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/messages/NoGamesError.qml" line="23"/>
+        <source>Pegasus couldn&apos;t find any games on your device. If you have not set up Pegasus yet, you can find the documentation here: &lt;i&gt;%1&lt;/i&gt;.&lt;br&gt;If you still see this message, make sure your config files are readable, exist in one of the expected locations and are in the expected format.</source>
+        <translation>Pegasus non trova alcun gioco nella tua periferica. Se non hai impostato ancora Pegasus, puoi trovare la documentazione qui: &lt;i&gt;%1&lt;/i&gt;.&lt;br&gt;If you still see this message, make sure your config files are readable, make sure that they are currently located in one of the expected locations and make sure that they are in the expected format.</translation>
```
Comment (mmatyas) [inline pegasus_it.ts]: Do you plan to translate the second half of this line?

**Review by mmatyas — CHANGES_REQUESTED** [review]
Thank you! I've added a few notes and questions.

File: pegasus_it.ts (hunk 1 above)
Comment (baggius) [reply pegasus_it.ts]: fixed

File: pegasus_it.ts (hunk 2 above)
Comment (baggius) [reply pegasus_it.ts]: done

File: pegasus_it.ts (hunk 3 above)
Comment (baggius) [reply pegasus_it.ts]: done

**Comment by mmatyas** [comment]
Looks good, thanks!

**Pull request merged**